web-mode.el

```diff
@@ -8353,6 +8353,8 @@ another auto-completion with different ac-sources (e.g. ac-php)")
     (cond
      ((or (null open-ctx) (null (plist-get open-ctx :pos)))
       (setq offset initial-column))
+     ((string-match-p "else$" prev-line)
```

Comment:
What about trailing whitespace?

Reply:
Wasn't an issue at my side, probably due to related whitespace removal settings, but I've tried adding support for it anyway (to the best of my elisp regexp abilities, which aren't nearly as good as my perl experience).